src/contributors.js

```diff
@@ -185,3 +185,7 @@ export const adilRafeek = {
   username: "adilrafeek"
 };
 
+export const DevanshDwivedi = {
+  name: "Devansh Dwivedi",
+  username: "port ghost"
```

Comment:
One more change. Sorry I missed this earlier.  Change it to your GitHub id "bottercode". Click the preview link posted by Netlify above. It currently shows undefined below your name.